docs/tutorials/cloud-init.md

```diff
@@ -70,75 +70,45 @@ Save it and close it.
 In PowerShell, run:
 
 ```{code-block} text
-> ubuntu2404.exe install --root
+> ubuntu2404.exe
 ```
 
-We skip the user creation since we expect cloud-init to do it.
+It will register a new Ubuntu-24.04 instance and wait for cloud-init to perform its initialisation tasks.
+That process can take several minutes, depending on the computer and network speeds.
 
 > If you want to be sure that there is now an Ubuntu-24.04 instance, run `wsl -l -v`.
 > Notice that the application is named `Ubuntu24.04LTS` but the WSL instance created is named `Ubuntu-24.04`.
 > See more about that naming convention in [our reference documentation](naming).
 
-## Check that cloud-init is running
-
-In PowerShell again run:
-
-```{code-block} text
-> ubuntu2404.exe run cloud-init status --wait
-```
-
-That will wait until cloud-init completes configuring the new instance we just created. When done, you should see an
-output similar to the following:
-
-```{code-block} text
-:class: no-copy
-..............................................................................
-..............................................................................
-..............................................................................
-...................
-status: done
-```
-
 ## Verify that it worked
 
-Restart the distro just to confirm that the changes in `/etc/wsl.conf` made by cloud-init will take effect.
-
-Terminate the running instance:
-
-```{code-block} text
-> wsl -t Ubuntu-24.04
-```
-
-This should output a message confirming that the instance has stopped:
-
-```{code-block} text
-:class: no-copy
-The operation completed successfully.
-```
-
-Now start the instance again:
-
-```{code-block} text
-> ubuntu2404.exe
-```
-
+When the setup is complete, the WSL instance's shell will be logged in as the user `jdoe`.
 You should see the standard welcome text:
 
 ```{code-block} text
 :class: no-copy
+Installing, this may take a few minutes...
+Installation successful!
 To run a command as administrator (user "root"), use "sudo <command>".
 See "man sudo_root" for details.
 
-Welcome to Ubuntu Noble Numbat (GNU/Linux 5.15.137.3-microsoft-standard-WSL2 x86_64)
+Welcome to Ubuntu 24.04.1 LTS (GNU/Linux 6.6.36.3-microsoft-standard-WSL2 x86_64)
 
  * Documentation:  https://help.ubuntu.com
  * Management:     https://landscape.canonical.com
  * Support:        https://ubuntu.com/pro
 
+ System information as of ter 01 out 2024 14:32:47 -03
+
+  System load:  1.64                Processes:             63
+  Usage of /:   0.2% of 1006.85GB   Users logged in:       0
+  Memory usage: 4%                  IPv4 address for eth0: 172.22.8.90
+  Swap usage:   0%
+
 
 This message is shown once a day. To disable it please create the
-/home/cn/.hushlogin file.
-jdoe@mib:~$
+/home/jdoe/.hushlogin file.
+jdoe@mib01:~$
 ```
 
 Once logged into the new distro instance's shell, verify that:
```

Comment:
Not something that needs to be addressed in this PR, and maybe a matter of taste :smile: but we could include a more compelling configuration setup in future. The functionality is demonstrated clearly but it is not obvious to me what the user is setting up with Octave and ginac-tools. For someone less familiar with auto-configuration or who is seeking a concrete use-case we could help them along with one or more examples for specific scenarios (e.g., data science, asset pipeline, programming in Go...).